src/strategies/go-yoshi.ts

```diff
@@ -195,10 +195,6 @@ export class GoYoshi extends BaseStrategy {
     );
     return releaseNotes.replace(/, closes /g, ', refs ');
   }
-
-  protected initialReleaseVersion(): Version {
-    return Version.parse('0.1.0');
-  }
```

Comment:
Not sure why the initial version number is different for different languages? That's kind of odd and doesn't match Stainless behaviour.